### ISSUE TITLE
[BUG] Make color of timer RED when less than 5 seconds are remaining.

### ISSUE THREAD

**Issue by Akanchha112** [SnowScriptWinterOfCode/QuickQuiz#36]
[BUG] Make the color of the timer RED when less than 5 seconds are remaining.

![image](https://github.com/SnowScriptWinterOfCode/QuickQuiz/assets/100962900/e92d5d76-7d67-4ed5-8a30-cd76d93b2a31)

**Comment by bj-jiwrajka** [comment]
@Akanchha112 isn't it exactly similar to #37

**Comment by Akanchha112** [comment]
> @Akanchha112 isn't it exactly similar to #37

No, in that you have to add animation, while in this you just have to change color.

**Comment by bj-jiwrajka** [comment]
> > @Akanchha112 isn't it exactly similar to #37
> 
> No, in that you have to add animation, while in this you just have to change color.

okay cool, assign it to me then 👌

**Comment by bj-jiwrajka** [comment]
@Akanchha112 please assign it to me
